tests/net/lib/zoap/src/main.c

```diff
@@ -64,61 +65,41 @@ static struct sockaddr_in6 dummy_addr = {
 	.sin6_family = AF_INET6,
 	.sin6_addr = IN6ADDR_LOOPBACK_INIT };
 
-static int test_build_empty_pdu(void)
+static void test_build_empty_pdu(void)
 {
 	u8_t result_pdu[] = { 0x40, 0x01, 0x0, 0x0 };
 	struct zoap_packet zpkt;
 	struct net_pkt *pkt;
 	struct net_buf *frag;
-	int result = TC_FAIL;
 	int r;
 
 	pkt = net_pkt_get_reserve(&zoap_pkt_slab, 0, K_NO_WAIT);
-	if (!pkt) {
-		TC_PRINT("Could not get packet from pool\n");
-		goto done;
-	}
+	zassert_not_null(pkt, "Could not get packet from pool");
```

Comment:
Messages in assertions should be written as if they were true: `zassert_not_null(pkt, "Get packet from pool")`. If `pkt` is `NULL`, ztest will then tell you that `the "get packet from pool" assertion failed`, which makes more sense. Otherwise, it's a double negative.

Reply:
Done. Thanks for the review